package.json

```diff
@@ -78,6 +87,7 @@
     "del": "^5.1.0",
     "electron": "^8.2.5",
     "electron-builder": "^22.6.0",
+    "electron-log": "^4.1.4",
```

Comment:
Probably want to remove dependencies from README and move it to a DEPENDENCIES.md file because might get to hard to manage listing all these dependencies

Reply:
This is a good point, however I don't think this is within the scope of this PR, so I'll create another PR just for reorganizing docs.